docs/rootless.md

```diff
@@ -45,6 +45,8 @@ penguin:231072:65536
   * Apparmor
   * Checkpoint
   * Overlay network
+  * Exposing SCTP ports
+* To expose a TCP/UDP port, the host port number needs to be set to >= 1024.
```

Comment:
oh; nit: looks like indentation is off here

Reply:
This is intentional. `Exposing SCTP ports` is an unsupported feature while `To expose a TCP/UDP port...` is a restriction

Reply:
oh! should've looked at the whole file; apologies